docs/content/specs/shared/_index.md

```diff
@@ -782,7 +868,7 @@ Example: `avm-res-compute-virtualmachine`
 
 #### ID: RMNFR2 - Category: Inputs - Parameter/Variable Naming
 
-A module **SHOULD** use the following standard inputs:
+A module **MUST** use the following standard inputs:
```

Comment:
Why would we enforce location? Some resources don't have that input, or require a hard-coded "global".

Reply:
Clarified